tests/Browser/CreateTaskTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Models\User;
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class CreateTaskTest extends DuskTestCase
+{
+    /**
+     * A Dusk test example.
+     *
+     * @return void
+     */
+    public function testExample()
+    {
+        $this->browse(function ($first) {
+            $first->loginAs(User::find(1))
+            ->visit('/task/create')
+            ->assertTitle('Laravel')
+            ->value('#name', 'Task #2')
+            ->select('category_id', '1')
```

Comment:
Where is the category id coming from? You should be using a separate "test" database and it should be migrated for each test. See https://laravel.com/docs/6.x/dusk#migrations.

Also, use Factories to create data needed for test (in this case you will need to create categories using Factory before creating the task in dusk).